man/foreign.doc

```diff
@@ -3019,7 +3018,22 @@ Discards the query, but does not delete any of the data created by the
 query.  It just invalidates \arg{qid}, allowing for a new call to
 PL_open_query() in this context.  PL_cut_query() may invoke cleanup
 handlers (see setup_call_cleanup/3) and therefore may experience
-exceptions.  If an exception occurs the return value is \const{FALSE}
+exceptions.\foot{An example of a handler is in
```

Comment:
Good.   I don't think we can have code blocks in a footnote though (and currently footnotes do not appear at all in the `?- help(Topic).`)

Reply:
Think I hit the resolved for some reason.   Yes, simplest is probably to delete this PR and recreate one with bibtex extries and this single fix in how exceptions from PL_cut_query() work.

P.s.   As you say, we keep the latest exception in case they are equal.  I wonder whether this is not wrong (and we should keep the first instead?)   This situation occurs if someone failed to clear an exception.  That is wrong code.  The other is that if handling the exception at some point raise a new exception.

Reply:
I'll have more to say about double exceptions in the Discourse forum -- I made this PR to confirm a few things. 
I'll re-do the PR (and will delete the 2 PRs I submitted), then will write some design questions in Discourse